.github/workflows/push-release.yml

```diff
@@ -0,0 +1,31 @@
+name: any-pr
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  push-master:
+    # Only run on any commit that features the release commit message syntax in the commit message
+    if: "contains(github.event.head_commit.message, 'release commit')"
+    name: Build, test and publish stable release
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.15.3]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm i
+    - run: npm run test
+    - run: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc 2> /dev/null
+        env:
+          NPM_TOKEN: ${{ secrets.NPMJS_TOKEN }}
```

Comment:
@thehenrytsai @csuwildcat can one of you add this to the repo secrets?

Reply:
Blocked from merging this PR until this secret is added